services/content-api.js

```diff
@@ -52,9 +52,22 @@ function getLegacyPage({ locale, path }) {
     });
 }
 
+function getListingPage({ locale, path }) {
+    return request({
+        url: `${API_URL}/v1/${locale}/listing`,
+        qs: {
+            path: path
+        },
+        json: true
+    }).then(response => {
+        return response.data.map(item => item.attributes).find(_ => _.path === path);
```

Comment:
This might need tweaking if/when we end up refactoring the API endpoint to work for multiple sections, but works okay for now.